Makefile

```diff
@@ -5,14 +5,24 @@ FULL_GOPATH=${GOPATH}/src/${FULL_NAME}
 
 default: test
 
-get:
-	go get "github.com/omeid/go-resources/cmd/resources"
+dependencies:	
 	go get -t -v ./...
 
-build: get
-	cd example/agent-ping && resources -output="resources.go" -var="Resources" -trim="../" resources/* ../schema/*
-	cd example/agent-pong && resources -output="resources.go" -var="Resources" -trim="../" resources/* ../schema/*
+build: dependencies	
 	go build -v ./...
 
+examples: dependencies
```

Comment:
You can but you will have to list explicitly the packages you want to build in the go build commands. It will be harder to maintain. However, I don't think this sdk is prone to have many more packages.

Why do you prefer to separate the 2 phases?